hack/update-labels.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TESTINFRA_ROOT=$(git rev-parse --show-toplevel)
+CONFIG=${CONFIG:-"${TESTINFRA_ROOT}/label_sync/labels.yaml"}
```

Comment:
These are pretty generic variables.. it's possible that there might be existing environment variables that could mess these up?

Reply:
good point, will prefix with `LABELS` since that's the suffix of these scripts